conf/defaults.ini

```diff
@@ -651,6 +651,8 @@ tls_client_key =
 tls_client_ca =
 use_pkce = true
 use_refresh_token = true
+# set to true by default, until we can map google groups to grafana org roles
+skip_org_role_sync = true
```

Comment:
I think it's safer to do this in the code where the config value is parsed. Otherwise this wouldn't work for people with custom configs who might have already set this value to `false`.